pkg/execution/svccatadmission/internaldns.go

```diff
@@ -0,0 +1,132 @@
+package svccatadmission
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"github.com/golang/glog"
+	"net/http"
+	"sync"
+
+	"github.com/atlassian/voyager/pkg/microsserver"
+	"github.com/atlassian/voyager/pkg/orchestration/wiring/internaldns/api"
+	sc_v1b1 "github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/v1beta1"
+	"github.com/pkg/errors"
+	admissionv1beta1 "k8s.io/api/admission/v1beta1"
+	ext_v1b1 "k8s.io/api/extensions/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// InternalDNSAdmitFunc checks existing DNS alias ownership via micros server API, for both InternalDNS Services and Ingress Resources
+func InternalDNSAdmitFunc(ctx context.Context, microsServerClient microsServerClient, scClient serviceCentralClient, admissionReview admissionv1beta1.AdmissionReview) (*admissionv1beta1.AdmissionResponse, error) {
+
+	glog.Infof("Received request for %v", admissionReview.Request)
```

Comment:
This information is already available in audit logs.

Reply:
This line was pushed by mistake. Will be fixed on next commits.

Reply:
fixed